app/templates/views/message-status.html

```diff
@@ -10,8 +10,8 @@
   <h1 class="font-body-2xl margin-bottom-3">Delivery status</h1>
 
   <p>Notify’s real-time dashboard lets you check the status of any message.</p>
-  <p>For <a class="usa-link" href="{{ url_for("main.security") }}">security</a>, this information is only available for seven days after a message has been sent. You can download a report, including a list of sent messages, for your own records.</p>
-  <p>This page describes the statuses you’ll see when you’re signed in to Notify.</p>
+  <p>For <a class="usa-link" href="{{ url_for('main.security') }}">security</a>, this information is only available for seven days after a message has been sent. You can download a report, including a list of sent messages, for your own records.</p>
+  <p>This page describes the statuses you'll see when you're signed in to Notify.</p>
 
   <!-- <p>If you’re using the Notify API, read our <a class="usa-link" href="{{ url_for('.documentation') }}">documentation</a> for a list of API statuses.<p>
```

Comment:
Do we need to keep these comments or can we remove?

Reply:
That's a good question - I think in the near future we can start removing some of these, they had been kept as a "just in case" to make sure we wouldn't need them.  As we start to go through replacing and updating things more significantly with the redesign effort though, that'll be a good opportunity to also remove them, or not carry them forward in the first place. :-)